modules/obr/dockerfiles/dockerfile.obrgalasabld

```diff
@@ -5,6 +5,6 @@ ARG tag
 FROM ${dockerRepository}/galasa-dev/obr-maven-artefacts:${tag}
```

Comment:
Could you pls add a description of what this docker image is trying to do ?
eg: 

An image which hosts a maven site with apache, where the maven site contains
all the maven artifacts from the OBR build, plus the galasactl command-line tool (all architectures/platforms).

Or something like that ?

Also, why do we need this ?
eg:
This is hosted, (where?) so that people can download the artifacts served up.

Or something like that ? Is that correct ? or is it the build process which needs it ?